prow/config/jobs/proxy-1.11.yaml

```diff
@@ -124,6 +124,25 @@ jobs:
   timeout: 4h0m0s
   types:
   - postsubmit
+- command:
+  - ../test-infra/tools/automator/automator.sh
+  - --org=istio
+  - --repo=proxy
+  - '--title=Automator: update envoy@$AUTOMATOR_SRC_BRANCH in $AUTOMATOR_ORG/$AUTOMATOR_REPO@$AUTOMATOR_BRANCH'
+  - --labels=auto-merge
```

Comment:
@istio/wg-networking-maintainers-data-plane or @istio/wg-product-security-maintainers Thoughts on whether or not we auto-merge? Otherwise I think this is ready.